src/client/snippets/paportal-liquid.code-snippets

```diff
@@ -87,5 +87,75 @@
 			"{% webform name:\"$1\" %}"
 		],
 		"description": "Fully renders a web form by name"
-	}
+	},
+    "1 column section": {
+        "prefix": "1-column-section",
+        "body": [
+            "<div class=\"row sectionBlockLayout\" style=\"text-align:left;min-height:374px;padding:8px;\"><div class=\"container\" style=\"display:flex;flex-wrap:wrap;\"><div class=\"col-md-12 columnBlockLayout\" style=\"padding:16px;margin: 60px 0;min-height:200px;\"></div></div></div>"
+        ],
+        "description": "Fully render a section component with 1 column"
```

Comment:
Can we review the description strings with Perry? "Fully render" doesn't make much sense

Reply:
Changed it to "Renders" instead of "Fully render". Similar to the already present snippets. I will also get it reviewed by Perry.

Reply:
Changed the descriptions strings to start with "Adds" for the time being.